packages/ag-charts-types/src/series/cartesian/barOptions.ts

```diff
@@ -22,7 +22,7 @@ export interface AgBarSeriesLabelOptions<TDatum, TParams> extends AgChartLabelOp
     padding?: PixelSize;
 }
 
-export interface AgBarSeriesItemStylerParams<TDatum>
+export interface AgBarSeriesItemStylerParams<TDatum = any>
```

Comment:
Ditto

Reply:
As above, to align with the grid convention for callbacks that can optionally be untyped

Reply:
Reverted